mcslock/Makefile

```diff
@@ -1,5 +1,5 @@
 all:
-	gcc -o tests -std=gnu11 -Wall -O2 mcslock.c tests.c
+	gcc -o tests -std=c11 -Wall -O2 mcslock.c tests.c
```

Comment:
Don't do this. We might use GNU extensions.

Reply:
OK, I'm revert it.